### ISSUE TITLE
No Use - Cobalt Dust (Mekanism Ores)

### ISSUE THREAD

**Issue by Danekjovax** [Freebi85/Dungeons-Dragons-Space-Shuttles#198]
I think this item needs to be fixed.  Currently no uses for this.  I think it should be meltable into ingot form (smeltery).

**Comment by Freebi85** [comment]
The only cobalt dust ingame is the one from rockhounding mod and can be melted in the smeltery as far as i can see. Dirty Cobalt dust is from meka ores and is required for the enrichment chambrer. Or i am missing somthing?

**Comment by Danekjovax** [comment]
The dust I'm referring to is the Cobalt Dust from MekanismOres (screenshot added).  JEI isn't giving me any uses for this dust, and I tried to place it into several furnaces and the smeltery, all of which do nothing to smelt it back into a cobalt ingot.
![2019-06-10_00 15 29](https://user-images.githubusercontent.com/10191173/59545466-8f5a5a00-8eec-11e9-930c-bd9eacffd293.png)

I have 20 pieces of the stuff, and don't recall how I got them at this time (it was a while ago, possibly through an earlier build that gave that dust).  I do notice that grinding cobalt ore/ingots now produce the Rockhounding version of the dust (albeit a counter-intuitive color) which does have the needed uses,.

I guess this just goes under the category of "old" item and I'll simply swap the 20 bits of it I have with the Rockhounding version in creative mode temporarily to fix it.

Lastly, JEI doesn't even list mekanism's cobalt dust... just the "dirty" version.  So I suspect this version was deprecated in an earlier build.  I just wish the cobalt dust in rockhounding shared the same cobalt color.

Thanks again.
